.github/actions/get-bootjdk/action.yml

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+name: 'Get BootJDK'
```

Comment:
Here and later, polishing: "BootJDK" -> "boot JDK"?

Reply:
I think we've mostly been using "BootJDK" as a specialized term in the build system, but we've not been very consistent about it: 
```
ihse@mercurius:~/git/jdk-ALT/open/make$ gr -i "Boot JDK" | wc
      61     673    5606
ihse@mercurius:~/git/jdk-ALT/open/make$ gr -i BootJDK | wc
      79     298    5947
```

but I can switch it out to `Boot JDK` if that makes you happier.

Reply:
Nevermind then.